### DISCUSSION TITLE
.gitignore

### DISCUSSION THREAD

**Discussion by hervehobbes** [adirh3/Fluent-Search#964] [Ideas]
Hello,
Thanks for a suck excellent program !
Is is possible to use the content of the .gitignore files for the indexer ?

**Comment by adirh3** [comment]
You will have to add the file types manually, you can do that in Settings -> Files -> Ignored paths and files, and use the `Files` column like this - `*.temp,*.g.*`

**Comment by hervehobbes** [comment]
Thank you for your answer.
I was hoping to have a setting to directly recognize this kind of file and use it ;-)

**Reply by adirh3** [reply]
Feel free to open a feature request for it in issues. As I am also a developer I would've liked it as well but there is a technical issue that makes it complicated to implement.
Either way FS should learn your file type usage and de-prioritzing temp files.
